pom.xml

```
@@ -10,18 +10,20 @@
 	</parent>
 	<groupId>io.jenkins.plugins</groupId>
 	<artifactId>inodes-monitor</artifactId>
-	<version>0.6-SNAPSHOT</version>
+	<version>${revision}</version>
 	<packaging>hpi</packaging>
 
 	<name>Inodes Monitor Plugin</name>
 	<description>
 	This plugin adds a column for inodes usage monitoring for Linux/Unix systems. Like for the existing disk space monitors,
 	it can help put automatically a node offline where not enough inodes are still available.
 	</description>
-	<url>https://wiki.jenkins.io/display/JENKINS/Inodes+Monitor+Plugin</url>
+	<url>https://plugins.jenkins.io/inodes-monitor/</url>
```

This is the location of the plugin documentation source.  That should be the README.md file of the plugin repository.  The plugins.jenkins.io site generation reads from this URL and creates the pages that are displayed on plugins.jenkins.io.

See [Migrate docs to GitHub](https://www.jenkins.io/doc/developer/tutorial-improve/migrate-documentation-to-github/) for the details.
```suggestion
	<url>https://github.com/${gitHubRepo}</url>
```